enos/modules/upgrade_client/scripts/verify_allocs.sh

```diff
@@ -48,17 +48,50 @@ done
 
 echo "Client $client_id at $CLIENT_IP is ready"
 
-# Quality: "nomad_alloc_reconect: A GET call to /v1/allocs will return the same IDs for running allocs before and after a client upgrade on each client"
-echo "Allocs found before upgrade $ALLOCS"
+allocs_count=$($ALLOCS |jq '[ .[] | select(.ClientStatus == "running")] | length')
+
+# Quality: "nomad_alloc_reconnect: A GET call to /v1/allocs will return the same IDs for running allocs before and after a client upgrade on each client"
+echo "$allocs_count allocs found before upgrade $ALLOCS"
+
+checkAllocsCount() {
+    local allocs
```

Comment:
These are all locals:

```suggestion
    local allocs running_allocs allocs_length
```